CodeOfConduct.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
```

Comment:
A link to this document from README.md or CONTRIBUTING.md would be useful for readers.